proposals/2021-05-gravity-bridge-deployment/README.md

```diff
@@ -0,0 +1,174 @@
+
+This oracle action will not be incentivized, nor will it be enforced with slashing. If validators making up more than 33% of the staked tokens do not participate in the oracle, new deposits and withdrawals will not be processed until those validators resume their oracle obligations.
+
+The oracle was designed to adhere to the Cosmos Hub validator security model without slashing conditions to ensure that a consensus failure on Ethereum does not affect operation of the Cosmos chain.
+
+### Slashing Conditions Spec
+
+https://github.com/cosmos/gravity-bridge/blob/main/spec/slashing-spec.md
+
+## How does it work?
+
+Gravity consists of 4 parts:
+
+- An Ethereum contract called Gravity.sol
+- The Gravity Cosmos module of Gaia
+- The orchestrator program which is run by Cosmos validators alongside the Gravity module
+- A market of relayers, who compete to submit transactions to Ethereum
+
+### Gravity.sol
+
+The Gravity Ethereum contract is a highly compact and efficient representation of weighted powers voting on Ethereum. It contains an Etheruem key from each Cosmos validator, as well as their voting power. This signer set is continuously updated as validation power changes on Cosmos, ensuring that it matches the current Cosmos validator set.
+
+Sending tokens, or updating the validator set, contained in Gravity.sol requires more than 66% of the total voting power to approve the action. In this way Gravity.sol mirrors Tendermint consensus on the Cosmos chain as closely as possible on Ethereum.
+
+### Gravity Cosmos Module
+
+The Gravity module governs and coordinates the bridge. Generating messages for the validators to sign with their Ethereum keys and providing these signatures to relayers who assemble and submit them to the Ethereum chain.
+
+### Orchestrator
+
+The Gravity bridge orchestrator performs all the external tasks the Gravity bridge requires for validators, which includes submission of signatures and submission of Ethereum events.
+
+While the Gravity module concerns itself with the correctness and consensus state of the bridge, the Orchestrator is responsible for locating and creating the correct inputs.
+
+### Market of Relayers
+
+Relayers are an unpermissioned role that observes the Cosmos chain for messages ready to be submitted to Ethereum.
+
+The relayer then packages the validators signatures into an Ethereum transaction and submits that transaction to the Ethereum blockchain. All rewards in the Gravity bridge design are paid to msg.sender on Ethereum. This means that relayers do not require any balance on the Cosmos side and can immediately liquidate their earnings into ETH while continuing to relay newer messages.
+
+## Security assumptions
+
+The Gravity bridge is designed with the assumption that the total amount of funds in Gravity.sol is less than the value of the validator set’s total staked tokens.
```

Comment:
Doesn't this assume 100% slashing? Right now slashing on the Hub is parameterized at 5% for equivocation and 0.01% for downtime:

```
$ gaiad query slashing params
slash_fraction_double_sign: "0.050000000000000000"
slash_fraction_downtime: "0.000100000000000000"
```

Will Gravity expose its own set of parameters here ?

Reply:
yes gravity will expose it's own set of parameters. I need to clarify some issues here as well.

Reply:
https://github.com/althea-net/cosmos-gravity-bridge/blob/main/spec/slashing-spec.md#gravslash-03-failure-to-sign-validator-set-update

Please see the second on `UnbondAttackSlashing` to cover this case.